spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/types/domain/HttpDataAddress.java

```diff
@@ -50,6 +50,8 @@ public class HttpDataAddress extends DataAddress {
     public static final String ADDITIONAL_HEADER = "header:";
     public static final String CONTENT_TYPE = "contentType";
     public static final String OCTET_STREAM = "application/octet-stream";
+    public static final String TRANSFER_CHUNKED = "transferChunked";
+    public static final String DEFAULT_TRANSFER_CHUNKED = Boolean.TRUE.toString();
```

Comment:
It is used by getProperty and getProperty supports only String as default value

Reply:
Looking at this further, it's odd that if a boolean property is not specified the default is true. IMO the switch should work as follows:

1. Default behavior should be to support the existing capability, streaming chunked if no value is specified. The PR appears to do this correctly.
2. If non-chunked behavior is required, a property should be set to "true". 

The configuration properties therefore should be inverted. This will also solve the issue with System.getProperty() since it will return a value or null which can then be parsed into a boolean directly.

Reply:
Does it feel more natural to say transferInOneGo instead of transferNonChunked (just picked one of the chunked antonyms) ?